### DISCUSSION TITLE
%-comparison is way off - not considering imported GA data?

### DISCUSSION THREAD

**Discussion by Swiss-Mac-User** [plausible/analytics#2867] [Anything Else]
The comparison vs previous period of my site are way off, e.g. when using „Current month“ view. Is this a bug in correlation with GA data import, where also native Plausible data has been tracked ?

I started tracking with Plausible last month (March) and it recorded „natively“ 550 pageviews. All the other data was imported from Google Analytics to keep the historical data.

It seems to me, that the data comparison vs last period (increase / decrease) does not take the imported data into account…

* Current month: 2.9k pageviews
* Last month: 4.1k pageviews
* Comparison: should be „-34%“
* _Instead it shows as_: „+450%“? (That’s the 550 pageviews natively recorded by Plausible in the last period)

![IMG_2158](https://user-images.githubusercontent.com/47624464/233833925-a2a4ca39-a8f4-4dc8-85f1-4e95706b8082.jpeg)
![IMG_2159](https://user-images.githubusercontent.com/47624464/233833928-f2e5e18c-f0c4-4459-8a0b-551089a5e753.jpeg)

**Comment by metmarkosaric** [comment]
thanks for reporting this @Swiss-Mac-User! our developers have managed to reproduce the issue. it is a bug with comparisons on date ranges that include both native and imported stats simultaneously. we'll work on fixing this

**Reply by Swiss-Mac-User** [reply]
Great to hear that you were able to find the root cause & a fix for it 🙏 Thanks @metmarkosaric

**Comment by metmarkosaric** [comment]
i've added this to the issues tab https://github.com/plausible/analytics/issues/2870

**Comment by metmarkosaric** [comment]
the fix for this is now live. thanks for reporting @Swiss-Mac-User!

**Reply by Swiss-Mac-User** [reply]
Can confirm that the %-comparison now shows the correct data 👍

A bit unfortunate IMHO that the comparison in the overview now only show actuals, but no more %-changes at a glance. But this is another story not related to the issue here 😉

**Reply by metmarkosaric** [reply]
Yeah, this is because we now have a dedicated comparison section. Do try it out for much more advanced comparison options and views
